qa/Vagrantfile

```diff
@@ -0,0 +1,27 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+require_relative "./platform_config.rb"
+
+Vagrant.configure(2) do |config|
+  platforms = PlatformConfig.new
```

Comment:
I'm talking about different bootstrapping needs, but you got your point, we can always extend it when necessary, what I don't want us to end up is with a very DRY vagrantfile, but _complicated_ to follow and understand.